designer/server/src/plugins/routes/api.ts

```diff
@@ -20,68 +20,64 @@ export const getFormWithId: ServerRoute = {
   options: {
     async handler(request, h) {
       const { id } = request.params
-      let formJson = newFormJson
       try {
         const response = await getPublished(id)
-        const { values } = JSON.parse(response)
+        const formJson = JSON.parse(response)
 
-        if (values) {
-          formJson = values
-        }
+        return h.response(formJson).type('application/json')
```

Comment:
Is calling `.type('application/json')` optional for existing objects?

Or maybe `JSON.parse()` is optional for already-stringified JSON?

```suggestion
        return h.response(formJson).type('application/json')
```

Reply:
nah not required, it's redundant. I can remove it.